docs/src/guides/run-ingest-on-terra.rst

```diff
@@ -0,0 +1,79 @@
+**************************
+Run Data Ingest on Terra
+**************************
+
+We have provided two pipelines for importing data into Terra:
+
+* GenBank Ingest - pull a public dataset and send them through our preprocessing scripts.
+* GISAID Ingest - pull a private dataset if a user has their own API key, account, and password.
+
+The pipelines were mainly motivated to provide access to our data pre-processing scripts. Currently, these are focused on pulling datasets for the ncov workflow and is roughly diagramed below:
+
+.. image:: ../images/terra-ingest.png
+
+
+Import ``ingest`` wdl workflow from Dockstore
+=============================================
+
+1. `Setup a Terra account <https://terra.bio/>`_
+#. Navigate to Dockstore: `ncov:wdl/genbank_ingest`_ or `ncov:wdl/gisaid_ingest`_ depending on if you wish to pull open (genbank) data or private (and have an gisaid api key) data.
+#. Top right corner, under **Launch with**, click on **Terra**
+#. Under "Workflow Name" set a name, such as ``genbank_ingest`` or ``gisaid_ingest``, and select your "Destination Workspace" in the drop down menu.
+#. Click button **IMPORT**
+#. In your workspace, click on the **WORKFLOWS** tab and verify that the imported workflow is showing a card
+
+.. _`ncov:wdl/genbank_ingest`: https://dockstore.org/workflows/github.com/nextstrain/ncov:wdl/genbank_ingest?tab=info
+.. _`ncov:wdl/gisaid_ingest`: https://dockstore.org/workflows/github.com/nextstrain/ncov:wdl/gisaid_ingest?tab=info
+
+Create Terra Variables for GISAID API
+=====================================
+
+If you are pulling GISAID data you must have your own API key. If you are pulling GenBank data (open), click on your imported "genbank_ingest" and skip to step 6.
+
+1. Navigate to your workspace on Terra
+#. On the **Data** tab, from the left menu click **Workspace Data**
+#. Create and fill in values for the following workspace variables:
+
+  +-----------------------------+----------------------------+-----------------------------------------------+
+  |Key                          | Value                      | Description                                   |
+  +=============================+============================+===============================================+
+  |GISAID_API_ENDPOINT          | url api enpoint value here | Provided by GISAID for your account           |
```

Comment:
Probably should standardize capitalization throughout. For example:

`url api enpoint value here` -> `URL API endpoint value here`

Note "enpoint" typo, too.